pkg/operator/cluster/ceph/mon/spec.go

```diff
@@ -102,6 +102,10 @@ func (c *Cluster) makeMonPod(config *monConfig, nodeName string) *v1.Pod {
 		podSpec.DNSPolicy = v1.DNSClusterFirstWithHostNet
 	}
 	c.placement.ApplyToPodSpec(&podSpec)
+	// don't set Pod (anti-)affinity because we have our own placement logic
```

Comment:
minor: could you either make this a 1-line comment or have consist spacing in the comment before the first word of the line?